app/src/components/Header/Header.styles.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+export const StyledHeader = styled.div`
+  margin: 0;
+  padding: 0;
+  padding-left: 60px;
+  width: 1440px;
+  height: 120px;
+  padding-top: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+`;
+export const StyledNavigation = styled.div`
+  display: flex;
+  width: 800px;
+  align-items: center;
+  justify-content: flex-start;
+`;
+export const StyledLogoNavLink = styled(NavLink)`
+  font-size: 28px;
+  color: black;
+  font-weight: 700;
+  text-decoration: none;
+`;
+export const StyledNavLink = styled(NavLink)`
+  margin-left: 30px;
+  font-size: 14px;
+  color: hsl(0, 0%, 75%);
```

Comment:
this should be taken from the theme colors

Reply:
already fixed